### PR TITLE
Lily/dcos oss 2086 update config headings

### PR DESCRIPTION
**Pull request by lilysamimi**
Problem: 
1. Labels across configuration flow, confirmation page and confirmation tab are inconsistent. 

Labels in configuration flow:

Service
Placement
Networking 
Volumes
Health Checks
Environment

Labels in Confirmation page and Confirmation tab: 

General
Placement
Network 
Storage
Environment Variables
Labels
Secrets
Health Checks

2. "key" and "value" labels in the Label section of the Confirmation page and Configuration tab are lower-case when they should be title-case. 

Solution:
1. Make labels consistent across configuration flow, confirmation page and configuration tab. 

All three pages should have the following labels for config flow: 

Service
Placement
Networking
Volumes
Health Checks
Environment

All three pages should have the following labels for confirmation screen and configuration tab: 

Service
Placement
Networking
Volumes
Health Checks
Environment Variables
Labels
Secrets

2. Make labels in the Label section title-case for Confirmation page and Configuration tab. 

Closes [DCOS_OSS-2086](https://jira.mesosphere.com/browse/DCOS_OSS-2086)

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/service-configuration/ServiceConfigDisplay.js [hunk 1]
```diff
@@ -38,9 +38,9 @@ const DEFAULT_DISPLAY_COMPONENTS = [
       ServicePlacementConstraintsConfigSection,
       ServiceNetworkingConfigSection,
       ServiceStorageConfigSection,
+      ServiceHealthChecksConfigSection,
```
Comment (ahoskins) [inline plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: 🤔 Why are these reordered?

Comment (ahoskins) [reply plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: Not a big deal, but looks like it was just accidentally committed.

**Comment by ahoskins** [comment]
run integration tests

File: plugins/services/src/js/service-configuration/ServiceConfigDisplay.js (hunk 1 above)
Comment (lilysamimi) [reply plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: Before Environment config was showing before Health Check config. I switched the order of the two to keep consistent with how we do it within the configuration flow.

**Comment by lilysamimi** [comment]
@bstavroulakis could you take a look at this and help update the strings? or tell me where to update them and I can try doing it myself.

File: plugins/services/src/js/service-configuration/ServiceConfigDisplay.js (hunk 1 above)
Comment (ahoskins) [reply plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: Ok sounds good 👍

**Pull request merged**